rendezvous/README.md

```diff
@@ -221,6 +262,7 @@ message Message {
     optional ResponseStatus status = 1;
     optional string statusText = 2;
     optional int64 ttl = 3; // in seconds
+    optional bytes challenge = 4;
```

Comment:
The more I think about it, the more it seems appropriate to include a `target_difficulty` here.

Reply:
Done in cd34ae9a3e998d28db16b0ee58df03b9339cb485.